Makefile

```diff
@@ -21,27 +21,35 @@ UNITDIR              = unit
 UNIT_TESTSDIR        = $(TESTS_DIR)/$(UNITDIR)
 EXAMPLES_DIR         = examples
 
-SRC := $(shell find $(SRCDIR) -name "*.c")
+SRC := $(wildcard $(SRCDIR)/*.c $(SRCDIR)/platform*/*.c)
 
 # Generate list of common test source files, only from tests/ dir. Hence '-maxdepth 1'.
 # These objects are shared between functional/ and unit/ test binaries.
-COMMON_TESTSRC := $(shell find $(TESTS_DIR) -maxdepth 1 -name "*.c")
-FUNCTIONAL_TESTSRC := $(shell find $(FUNCTIONAL_TESTSDIR) -name "*.c")
+COMMON_TESTSRC := $(wildcard $(TESTS_DIR)/*.c)
```

Comment:
The fact that `wildcard` does not recurse under sub-dirs implicitly gives us the `-maxdepth 1` functionality of the `find` command. (That's good for us!)

Reply:
Cool, but update the comment to eliminate the reference to "-maxdepth 1".